components/brave_wallet_ui/components/extension/assets-panel/index.tsx

```diff
@@ -150,9 +174,14 @@ const AssetsPanel = (props: Props) => {
               : ''
           }
           token={token}
+          spotPrice={
+            spotPriceRegistry && !isLoadingSpotPrices && !isFetchingSpotPrices
+              ? getTokenPriceAmountFromRegistry(spotPriceRegistry, token)
+              : Amount.empty()
+          }
```

Comment:
this will always cause a ren-render due to a new `Amount` object instance being created.